### ISSUE TITLE
AckDeadline does not work even after subscription creation with v1.SubscriptionClient

### ISSUE THREAD

**Issue by muryginm** [googleapis/nodejs-pubsub#81]
In my app I really need to configure message ackDeadline. Because of [that issue](https://github.com/googleapis/nodejs-pubsub/issues/6) I am not able to do it using `createSubscription`. So I found out a way to create subscription with specific ackDeadlineSeconds using `v1.SubscriptionClient`. But client library does not care about that setting.

#### Environment details

  - OS: Ubuntu 16.04
  - Node.js version: 8.9.4
  - npm version: 5.5.1
  - @google-cloud/pubsub version: 0.16.4

I created a repository with code to reproduce this issue.

#### Steps to reproduce
  1. Clone repository https://github.com/muryginm/google-cloud-pubsub-issues
  2. `npm install`
  3. `GOOGLE_PROJECT=your-project-id GOOGLE_APPLICATION_CREDENTIALS=path-to-credentials npm run ack-deadline`
  4. In most cases the message was resent after ~60s even when I have ackDeadlineSeconds 180s.

In source code I did the following:
 1. Created topic
 2. Created subscription with ackDeadlineSeconds using `v1.SubscriptionClient`
 3. After step 2 I really had a subscription with custom ackDeadlineSeconds (I checked it using console.cloud.google.com and the command `gcloud pubsub subscriptions describe name`)
 4. Tracked the time between `message` events.

How can I achieve the correct behaviour of ackDeadlineSeconds feature?

Thanks!

**Comment by callmehiphop** [comment]
Currently we don't allow the ack deadline to be modified. The Subscription object has an auto-leasing feature that will automatically extend the deadline on any messages waiting to be acked. Even if we allowed the deadline to be set, the current code would overwrite it and your message would not get redelivered until your subscription object was closed.

I'm going to close this issue because it looks to be a duplicate of https://github.com/googleapis/nodejs-pubsub/issues/6. I'll be happy to re-open if you think that's not the case.
